### ISSUE TITLE
404 OpenURI::HTTPError when trying to create the rails app

### ISSUE THREAD

**Issue by jramiresbrito** [lewagon/rails-templates#118]
Steps applied:

```
rails new \
  --database postgresql \
  --webpack \
  -m https://raw.githubusercontent.com/lewagon/rails-templates/rails-six/devise.rb flash-v2
```

Expected behavior:

```
A new rails application
```

Current behavior:

```
404 Not Found. (OpenURI::HTTPError)
```

Traceback:

```
apply  https://raw.githubusercontent.com/lewagon/rails-templates/rails-six/devise.rb

Traceback (most recent call last):
	30: from /home/joao/.rbenv/versions/2.6.5/bin/rails:23:in `<main>'
	29: from /home/joao/.rbenv/versions/2.6.5/bin/rails:23:in `load'
	28: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/railties-6.0.3.1/exe/rails:10:in `<top (required)>'
	27: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:72:in `require'
	26: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:72:in `require'
	25: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/railties-6.0.3.1/lib/rails/cli.rb:18:in `<top (required)>'
	24: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/railties-6.0.3.1/lib/rails/command.rb:46:in `invoke'
	23: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/railties-6.0.3.1/lib/rails/command/base.rb:69:in `perform'
	22: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor.rb:392:in `dispatch'
	21: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/invocation.rb:127:in `invoke_command'
	20: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/command.rb:27:in `run'
	19: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/railties-6.0.3.1/lib/rails/commands/application/application_command.rb:26:in `perform'
	18: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/base.rb:485:in `start'
	17: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/group.rb:232:in `dispatch'
	16: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/invocation.rb:134:in `invoke_all'
	15: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/invocation.rb:134:in `map'
	14: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/invocation.rb:134:in `each'
	13: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/invocation.rb:134:in `block in invoke_all'
	12: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/invocation.rb:127:in `invoke_command'
	11: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/command.rb:27:in `run'
	10: from (eval):1:in `apply_rails_template'
	 9: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/railties-6.0.3.1/lib/rails/generators/app_base.rb:166:in `apply_rails_template'
	 8: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/thor-1.0.1/lib/thor/actions.rb:222:in `apply'
	 7: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/2.6.0/open-uri.rb:35:in `open'
	 6: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/2.6.0/open-uri.rb:736:in `open'
	 5: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/2.6.0/open-uri.rb:165:in `open_uri'
	 4: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/2.6.0/open-uri.rb:224:in `open_loop'
	 3: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/2.6.0/open-uri.rb:224:in `catch'
	 2: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/2.6.0/open-uri.rb:226:in `block in open_loop'
	 1: from /home/joao/.rbenv/versions/2.6.5/lib/ruby/2.6.0/open-uri.rb:756:in `buffer_open'
/home/joao/.rbenv/versions/2.6.5/lib/ruby/2.6.0/open-uri.rb:378:in `open_http': 404 Not Found (OpenURI::HTTPError)
```

**Comment by rahulkeerthi** [comment]
Hey @jramiresbrito (and everyone else!) 
See @andrerferrer's fix to the README in #117 
Recent merge changed the folder structure so it's now `master` rather than `rails-six`

FYI @Eschults

**Comment by jramiresbrito** [comment]
> Hey @jramiresbrito (and everyone else!)
> See @andrerferrer's fix to the README in #117
> Recent merge changed the folder structure so it's now `master` rather than `rails-six`
> 
> FYI @Eschults

Worked. I'll close the Issue since probably tomorrow the readme will be merged.
